### ISSUE TITLE
Allow specifying AltServer IP Address

### ISSUE THREAD

**Issue by tech234a** [rileytestut/AltStore#163]
I would be helpful if we could specify an AltServer IP address within AltStore, and an iPhone IP address within AltServer (Windows) because my devices can never find each other on their own. Xcode has a feature like this when you right click on the device name in the Devices and Simulators window and select "Connect via IP Address...".

**Comment by kylemart** [comment]
I would also be interested in this. In my case, I have a VPN server that allocates devices within a different subnet from my AltServer VM (10.0.0.* vs 192.168.1.*). I'd like to be able to VPN into my home network, and sync my iPad with AltServer. However, whenever I try to do this, the app is unable to find my server. 

Still looking through the implementation, but it seems that the app isn't searching for devices on the LAN subnet. If I could specify that my server is at IP 192.168.1.123, that would probably fix the issue for me rather easily.

**Comment by kylemart** [comment]
@noah978 - Do you know if designating a static IP in the app would be enough to resolve my situation? Or would there be constraints with iTunes (and having the server and client in different subnets)?

**Comment by kylemart** [comment]
Seems this contains logic related to having the app find AltServer: https://github.com/rileytestut/AltStore/blob/master/AltStore/Server/ServerManager.swift

**Comment by hethhhhh** [comment]
I have the same issue, can we see IP address field? or any other way to connect to AltStore over VPN. <3

**Comment by tajetaje** [comment]
Here here, I have a zerotier network setup and this would allow AltServer and AltJIT to theoretically work permanently

**Comment by dennore** [comment]
yes, please add an optional field for hostname/ip so that we can use it with VPN.

Great work!
